tests/Umbraco.Tests.Integration/Testing/UmbracoIntegrationTest.cs

```diff
@@ -84,6 +84,8 @@ public void Setup()
         {
             Services.GetRequiredService<IUmbracoContextFactory>().EnsureUmbracoContext();
         }
+
+        CleanupViewFiles();
```

Comment:
This doesn't seem related to the main changes in this PR.  Did you find an issue that needed solving here?